pkg/mads/v1/service_test.go

```diff
@@ -457,5 +458,119 @@ var _ = Describe("MADS http service", func() {
 			Expect(discoveryRes.VersionInfo).ToNot(BeEmpty())
 			Expect(discoveryRes.Resources).To(HaveLen(2))
 		})
+
+		It("should allow specifying the fetch timeout", func() {
```

Comment:
nit: can we have one more test with an invalid value for the query param?

Reply:
Added in ac0460d